Documentation/ReleaseNotes.md

```diff
@@ -38,13 +38,67 @@ If importing the [Mixed Reality Toolkit NuGet packages](MRTKNuGetPackage.md), th
 
 ### Upgrading projects to 2.4.0
 
-#### Custom controller classes
+The 2.4.0 release has some changes that may impact application projects. Breaking change details, including mitigation guidance, can be found in the [**Updating 2.3.0 to 2.4.0**](Updating.md#updating-230-to-240) article.
 
-Custom controller classes previously had to define `SetupDefaultInteractions(Handedness)`. This method has been made obsolete in 2.4, as the handedness parameter was redundant with the controller class' own handedness. The new method has no parameters. Additionally, many controller classes defined this the same way (`AssignControllerMappings(DefaultInteractions);`), so the full call has been refactored down into `BaseController` and made an optional override instead of required.
+> [!NOTE]
+> At this time, it is not supported to switch between using .unitypackage files and NuGet.
+
+**Updating using .unitypackage files**
+
+For the smoothest upgrade path, please use the following steps.
```

Comment:
It does feel like we should put this in one place and keep it updated between versions. It's mostly copied and pasted between each version section in this doc

Reply:
Let's do that as a separate PR :)